ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+// import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: '#FAFAFA',
+    height: 200
+  },
+  centered: {
+    width: '100%'
+  },
+  title: {
+    fontSize: 14
+  }
+
+}))
+
+function FeatureCard (props: FeatureCardProps): JSX.Element {
+  const classes = useStyles()
+
+  return (
+    <Card className={`${classes.root}`} variant="outlined">
```

Comment:
The specific of this might change later when the link is added, but what do you think about adding `tabIndex={0}` just to make these keyboard accessible so we remember that's needed later?

Reply:
Good idea, thanks

Reply:
Added `tabIndex={props.ordinalty}`